public/app/percona/shared/components/Form/MultiSelectField/MultiSelectField.tsx

```diff
@@ -2,16 +2,35 @@ import React, { FC } from 'react';
 import { MultiSelect } from '@grafana/ui';
 import { MultiSelectCommonProps } from '@grafana/ui/src/components/Select/types';
 import { withSelectStyles } from '../withSelectStyles/withSelectStyles';
-import { Label } from '../Label';
-import { MultiSelectFieldProps } from './MultiSelectField.types';
+import { Label } from '@percona/platform-core';
+import { LabeledFieldProps } from '@percona/platform-core/dist/shared/types';
```

Comment:
If we really need this type we should export it properly in core-ui to avoid this import.